api/nodes/serializers.py

```diff
@@ -110,6 +118,7 @@ def get_pointers_count(self, obj):
         return len(obj.nodes_pointer)
 
     def create(self, validated_data):
+        validated_data.pop('type')
```

Comment:
@brianjgeiger `type` is special in that it isn't an attribute of the stored objects, so it shouldn't be present in `validated_data`. This prevents bugs when we do

```python
# id and type shouldn't be in validated_data
for attr, value in validated_data.items():
    setattr(model_instance, attr, value)
model_instance.save()
```